### ISSUE TITLE
Error while saving

### ISSUE THREAD

**Issue by Bartheleway** [daishi4u/ForceCode#110]
**Describe the bug**
It appears that sometimes when saving I got this error message : illegal value for line

This may happen when I have a dependent class which is invalid, for example the last case I found was having this error while saving in the SF dev console :
```
Dependent class is invalid and needs recompilation:
 Class [CLASSNAME] : Dependent class is invalid and needs recompilation:
 Class [CLASSNAME] : Dependent class is invalid and needs recompilation:
 Class [CLASSNAME] : Method does not exist or incorrect signature: void [METHOD_NAME](List<sObject>, Boolean) from the type [CLASSNAME]
```

And sometimes the extension is giving me the "dependent class" error. My assumption is that it fails to parse the given error message due to a special character.

**Expected behavior**
I should be able to have this message in VS Code.

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: chrome
 - Version: 3.9.3

**Comment by daishi4u** [comment]
Update to the marketplace version, as I am pushing to celador's branch now. I believe there was a fix for this a few versions ago, but I also put in an extra check for the upcoming 3.9.7 release.

**Comment by tomas-migra** [comment]
@daishi4u should we than stop using this repository for raising issues?

**Comment by daishi4u** [comment]
Yeah you can put issues on the marketplace repo at celador/ForceCode. Thanks!
